third_party/2and3/yaml/cyaml.pyi

```diff
@@ -18,3 +20,25 @@ class CLoader(CParser, SafeConstructor, Resolver):
 
 class CSafeLoader(CParser, SafeConstructor, Resolver):
     def __init__(self, stream: Union[str, bytes, _Readable]) -> None: ...
+
+class CDangerLoader(CParser, Constructor, Resolver): ...  # undocumented
+
+class CEmitter:
+    def __init__(self, stream: IO[Any], canonical: Any = ..., indent: Optional[int] = ...,
```

Comment:
You should use `Optional[]` for canonical, allow_unicode, and explicit_(start|end) even though the type is `Any`: it's helpful to the reader, and it'll trigger `None` safety checks in both mypy and pytype.